send_webhook_payload.py

```diff
@@ -0,0 +1,79 @@
+# send_webhook_payload.py
+# This action extracts metadata from a GitHub Actions event and sends it via HTTP to a database.
+# Note: For now, this action only supports pull request events.
+# Inputs:
+# os.environ['INPUT_COLLECTOR_ENDPOINT']
+# os.environ['INPUT_COLLECTOR_TOKEN']
+# os.environ['INPUT_JIRA_KEYS']
+# os.environ['INPUT_EVENT']
+
+# imports
+import json
+import os
+import re
+import requests
+
+
+def get_pull_request_info(event_json):
+    pull_request_info = {}
+    pull_request_info['number'] = event_json["pull_request"]["number"]
+    pull_request_info['url'] = event_json["pull_request"]["url"]
+    pull_request_info['title'] = event_json["pull_request"]["title"]
+    pull_request_info['created_at'] = event_json["pull_request"]["created_at"]
+    pull_request_info['description'] = event_json["pull_request"]["body"]
+    pull_request_info['author'] = event_json["pull_request"]["user"]["login"]
+    return pull_request_info
+
+
+if __name__ == '__main__':
+    # read in env vars
+    collector_endpoint = os.environ['INPUT_COLLECTOR_ENDPOINT']
+    collector_token = os.environ['INPUT_COLLECTOR_TOKEN']
+    jira_keys = os.environ['INPUT_JIRA_KEYS']
+    event = os.environ['INPUT_EVENT']
+    if collector_endpoint == "" or collector_token == "" or jira_keys == "" or event == "":
+        print(f"collector_endpoint: {collector_endpoint}")
+        print(f"collector_token: {collector_token}")
+        print(f"jira_keys: {jira_keys}")
+        print(f"event: {event}")
+        raise ValueError("None of the inputs should be null.")
+
+    # extract all JIRA tickets
+    jira_tickets = []
+    for key in jira_keys.split(","):
+        regex_string = r"\b" + key + r"-\d+"
+        matches = re.findall(regex_string, event)
+        if matches:
+            jira_tickets.append(matches)
+    # flatten list of lists, if needed
+    all_jira_tickets = [item for sublist in jira_tickets for item in sublist]
+
+    # read in event data into json for remaining queries
+    event_json = json.loads(event)
+
+    # construct payload
+    #  extract general metadata
+    payload = {}
+    payload['repo_org'] = event_json["repository"]["full_name"].split("/")[0]
+    payload['repo_name'] = event_json["repository"]["name"]
+    payload['repo_url'] = event_json["repository"]["html_url"]
+    payload['jira_tickets'] = all_jira_tickets
```

Comment:
Can we add the version of the github action to the payload?

Reply:
It's not present in the runner's env vars, apparently: https://docs.github.com/en/actions/learn-github-actions/environment-variables

And I don't htink it's in the `github` context accessible via the workflow file, either: https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

@osterman I'm guessing the easiest way to deal with this would be to force users to manually add the version as an input to the workflow whenever they call it. Do you want to do that?